Configs/ec/data/data_captions.yaml

```diff
@@ -0,0 +1,6 @@
+# Loading and saving data:
```

Comment:
In this, and similar files, use variables? https://omegaconf.readthedocs.io/en/latest/usage.html#variable-interpolation

For example: 
```
base_dir: ./DataLink/captioning_new_new
train_captions: ${base_dir}/en_captions_train.jsonl
```

Reply:
done!